dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/annotations/processors/permissions/extractors/Empty.java

```diff
@@ -0,0 +1,12 @@
+package edu.harvard.iq.dataverse.annotations.processors.permissions.extractors;
+
+import edu.harvard.iq.dataverse.persistence.DvObject;
+
+public class Empty implements DvObjectExtractor {
+
+    @Override
+    public DvObject extract(Object input) {
+        throw new IllegalStateException("This extractor should have not been called. Probably erroneous configuration" +
+                "of permission annotations for service method.");
```

Comment:
I'm thinking if default extractor shouldn't be the one that simply returns the input (no extraction). That way it wil be one less if in processor